dashboard/test/ui/features/step_definitions/account_steps.rb

```diff
@@ -204,3 +204,10 @@ def pass_time_for_user(name, amount_of_time)
   user.permission = UserPermission::AUTHORIZED_TEACHER
   user.save!
 end
+
+And(/^I give user "([^"]*)" universal instructor permission$/) do |name|
+  require_rails_env
```

Comment:
just a heads up that this is call is very expensive (30-60sec). it's fine to follow this existing pattern for this PR, but if you are going to be adding a lot of tests which use this step then it could be worth adding a new API to the test_controller.rb instead.

Reply:
Can you give an example?

Reply:
* https://github.com/code-dot-org/code-dot-org/blob/c2b301515eabae5be81f167dfa53cd4a45261031/dashboard/test/ui/features/step_definitions/levelbuilder_steps.rb#L14-L17
* https://github.com/code-dot-org/code-dot-org/blob/f90b6f8af847d503f0fdd9a670775b51e6ebca74/dashboard/app/controllers/test_controller.rb#L66

Reply:
oh, actually here is a much better example:
* https://github.com/code-dot-org/code-dot-org/blob/c2b301515eabae5be81f167dfa53cd4a45261031/dashboard/test/ui/features/step_definitions/levelbuilder_steps.rb#L10-L11
* https://github.com/code-dot-org/code-dot-org/blob/f90b6f8af847d503f0fdd9a670775b51e6ebca74/dashboard/app/controllers/test_controller.rb#L12-L17

It appears to rely on current_user, so it seems like your user would have to be signed in already in a previous step in the UI test for this API to work.

Reply:
one more note, it seems that the test controller has some logic so that a route gets auto-created for any method you add to it.

Reply:
Ok great thanks!